chart/templates/replicated-deployment.yaml

```
@@ -47,6 +47,9 @@ spec:
           readOnly: true
           subPath: config.yaml
         env:
+        {{- with .Values.extraEnv }}
+        {{ toYaml . | nindent 8 }}
```

```suggestion
        {{- toYaml . | nindent 8 }}
```

small suggestion so that we don't end up with an extra newline:
```
        env:
        
        - name: HTTP_PROXY
          value: http://localhost:3000
        - name: HTTPS_PROXY
          value: http://localhost:3000   
```

---

Great, thanks!